lib/timers.js

```diff
@@ -136,17 +143,14 @@ function setTimeout(callback, after, arg1, arg2, arg3) {
       break;
     default:
       args = [arg1, arg2, arg3];
-      for (i = 5; i < arguments.length; i++) {
+      for (let i = 5; i < arguments.length; i++) {
```

Comment:
I ran the benchmark for whether use let in for-loop or not before. If I remember correctly, the result is `let` out for-loop is faster than `let` in it in many cases

Reply:
Could you please run the benchmark for this?